eng/Version.Details.xml

```diff
@@ -57,360 +57,356 @@
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
+      <Sha>5f72b6f951b3d9db93b46454e78d9aa3c7f6a70a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.TypeNameHelper.Sources" Version="5.0.0-alpha.1.20079.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.TypeNameHelper.Sources" Version="5.0.0-preview.1.20106.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
+      <Sha>5f72b6f951b3d9db93b46454e78d9aa3c7f6a70a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.ValueStopwatch.Sources" Version="5.0.0-alpha.1.20079.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.ValueStopwatch.Sources" Version="5.0.0-preview.1.20106.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
+      <Sha>5f72b6f951b3d9db93b46454e78d9aa3c7f6a70a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.WebEncoders" Version="5.0.0-alpha.1.20079.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Extensions.WebEncoders" Version="5.0.0-preview.1.20106.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
+      <Sha>5f72b6f951b3d9db93b46454e78d9aa3c7f6a70a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Internal.Extensions.Refs" Version="5.0.0-alpha.1.20079.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Internal.Extensions.Refs" Version="5.0.0-preview.1.20106.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
+      <Sha>5f72b6f951b3d9db93b46454e78d9aa3c7f6a70a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.JSInterop" Version="5.0.0-alpha.1.20079.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.JSInterop" Version="5.0.0-preview.1.20106.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
       <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>275e691f7e575f208290d1cbb8cb450f4a3a85d6</Sha>
+      <Sha>5f72b6f951b3d9db93b46454e78d9aa3c7f6a70a</Sha>
     </Dependency>
-    <Dependency Name="Mono.WebAssembly.Interop" Version="5.0.0-preview3.1.20079.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
```

Comment:
FYI only: The other option would have been to remove the `CoherentParentDependency` attribute and add `Pinned="true"`. Moving the package to the 'Manual' section is probably better when we don't expect the package to come back.